README.md

```diff
@@ -118,13 +128,15 @@ In this example we just pass the jdbc connection url and set ssl to false. The f
              },
 ```
 In the final section of the request body is where we set the final configuration properties.
-`metadataSourceQualifiedName` specifies the Egeria qualified name of the entity that represents the Software Server entity represents the postgres server.
-If you have already created and entity to represent the postgres server simply enter the qualified name here. If you haven't already created the server
+`metadataSourceQualifiedName` specifies the Egeria qualified name of the entity that represents the Software Server entity represents the Postgres server.
+If you have already created and entity to represent the Postgres server simply enter the qualified name here. If you haven't already created the server
 entity, one will be automatically created by the integration daemon.
+The 'refreshTimeInterval' specifies the time lapse between refresh cycles in minutes.
+The 'permittedSynchronization' specifies the Integration Daemons thread synchronization model.
 
 ```shell
 
-             "metadataSourceQualifiedName" : "postgresHostServer",
+             "metadataSourceQualifiedName" : "PostgresHostServer",
```

Comment:
you may want this to be lowercase as a example, but it's unimportant as just an example

Reply:
leave as is, as it's just an identifier